sites/partners/src/components/applications/ApplicationsColDefs.ts

```diff
@@ -492,7 +492,7 @@ export function getColDefs(maxHouseholdSize: number, countyCode: string) {
 
   const householdCols = []
 
-  for (let i = 0; i < maxHouseholdSize; i++) {
+  for (let i = 0; i < maxHouseholdSize - 1; i++) {
```

Comment:
Previously on the applications table in partners, we were always showing an additional set of household member columns that never had data